app/background.cc

```diff
@@ -18,16 +18,17 @@ struct background_t {
     int frm;
     double fps;
     cv::Mat raw;
+    cv::Mat thm;
     std::thread thr;
     std::mutex mux;
+    std::mutex mtm;
```

Comment:
Same here: What's the difference between those two mutexes?

Reply:
as above - I'll use proper names for variables!